go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -1141,6 +1141,12 @@ func TestLoadDataErrors(t *testing.T) {
 	enginetest.TestLoadDataErrors(t, h)
 }
 
+func TestBinlog(t *testing.T) {
```

Comment:
We have existing tests for binlog features in the `binlogreplication` package (which run in a separate CI job since the binlog tests take a while to run). 

Instead of putting these in `enginetest`, it probably makes more sense to keep them all together in the tests in the `binlogreplication` package. Otherwise, it's a bit confusing for people who come and read the code later when they try to figure out the difference between `TestBinlog` and the separate binlog tests in the `binlogreplication` package.